articles/roadmap-preview-january-2025.md

```diff
@@ -0,0 +1,32 @@
+# Roadmap preview, January 2025
+
+<div purpose="embedded-content">
+   <iframe src="https://www.youtube.com/embed/G5W5pjzOfVM" allowfullscreen></iframe>
+</div>
+
+The Fleet roadmap is set for the first three months of 2025. Watch the video above for a walkthrough, or continue reading for the highlights.
+
+In the next 3 months, Fleet will ship...
+- 🛡️ Integration with Microsoft compliance tools
+- 🦾 Read-only GitOps mode
+- 💝 Fleet-maintained apps for Windows
+- ☑️ Integration with DigiCert
+- 🎡 One queue for MDM commands, software, and scripts
+- 📱 Android MDM for personal devices (BYOD)
+- 🔍 More data for detection & response and threat hunting
+- 🔄 Auto-patch software without writing custom policies
+- 🧑‍💼 Identify provider host vitals
+
+Big opportunities that Fleet is building towards in the near future (next 180 days):
+- 🍏 Account-based user enrollment for personal devices (BYOD)
+- 🗓️ Native patching for apps and OS during maintenance windows
+- 🤖 AI-generated osquery queries
+
+Any feedback or a questions? You can find us where we hang out in the [osquery // #fleet Slack channel](https://chat.osquery.io/c/fleet).
```

Comment:
This is the security-focused channel.  Let's send folks to the "Ask the community" page: https://fleetdm.com/support

@noahtalerman @Drew-P-drawers